webdriver/tests/bidi/script/get_realms/get_realms.py

```diff
@@ -0,0 +1,178 @@
+import pytest
+
+from webdriver.bidi.modules.script import ContextTarget
+
+from ... import any_string, recursive_compare
+
+PAGE_ABOUT_BLANK = "about:blank"
+
+
+@pytest.mark.asyncio
+async def test_payload_types(bidi_session):
+    result = await bidi_session.script.get_realms()
+
+    recursive_compare(
+        [
+            {
+                "context": any_string,
+                "origin": any_string,
+                "realm": any_string,
+                "type": any_string,
+            }
+        ],
+        result,
+    )
+
+
+@pytest.mark.asyncio
+async def test_calling_twice(bidi_session):
```

Comment:
(just a matter of taste, feel free to discard)
```suggestion
async def test_realm_is_consisnent_when_calling_twice(bidi_session):
```

Reply:
renamed ✅